fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -198,6 +219,16 @@ class CDUFlightPlanPage {
                 }
             }
 
+            const constraintType = wp ? CDUVerticalRevisionPage.constraintType(mcdu, wp) : WaypointConstraintType.Unknown;
+            if (constraintType === WaypointConstraintType.CLB) {
+                useTransitionAltitude = true;
+            } else if (constraintType === WaypointConstraintType.DES) {
+                useTransitionAltitude = false;
+            } else if (tocIndex >= 0) {
+                // FIXME Guess because VNAV doesn't tell us whether altitudes are climb or not \o/
```

Comment:
soon™

Reply:
There's a reason I pinged you for review. 😁 I'd appreciate if you can follow this one up when you have a chance some time 👍 .